duplicated-projects.md

```diff
@@ -0,0 +1,51 @@
+## Title
+Duplicated Projects
+
+## Patlet
+Forthcoming ...
+
+## Problem
+* Multiple teams are doing inner source, but they don't want to work together.
+* Multiple teams are working on the same product.  How do they merge?
+
+## Story (Optional)
+e.g. Splunk. 
+
+* 4 unique products have significant overlap.  Discovered after inner sourcing.
```

Comment:
```suggestion
* 4 unique products have significant overlap.  Discovered after opening them up as InnerSource projects.
```

Reply:
I worked this in via a separate commit. Thanks for the fix @lenucksi !